src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.Liquid.Converter.nuspec

```diff
@@ -33,6 +33,6 @@
   <files>
     <file src="bin\Release\netcoreapp3.1\Microsoft.Health.Fhir.Liquid.Converter.dll" target="lib\netcoreapp3.1" />
     <file src="bin\Release\netcoreapp3.1\Microsoft.Health.Fhir.TemplateManagement.dll" target="lib\netcoreapp3.1" />
-    <file src="..\..\data\DefaultTemplates.tar.gz" target="contentFiles\any\netcoreapp3.1" />
+    <file src="..\..\bin\DefaultTemplates.tar.gz" target="contentFiles\any\netcoreapp3.1" />
```

Comment:
Is the `bin` a automatically generated top-level directory?

Reply:
Yes, `bin` is generated only when running the pipeline. It is used to host data/binaries that are archived/published in the pipeline (Line 66/55 in release.yml). We specify the location of these files in nuspec and later they'll be packed into NuGet packages.